packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -441,11 +394,49 @@ export default class Tile extends Surface {
       );
     }
   }
+  _removeProgressBar() {
+    this._Content.patch({ ProgressBar: undefined });
+    this._updateMetadata();
+  }
+  /* ------------------------------ Metadata  ------------------------------ */
+
+  get _shouldShowMetadata() {
+    return this._persistentMetadata || this._isFocusedMode;
+  }
+
+  get _isInsetMetadata() {
+    return this._metadataLocation === 'inset';
+  }
+  // called in animateMetadata
```

Comment:
can this comment get moved inside the method or something? At a minimum, let's add an empty line after line 409.

Reply:
I'll remove the comment. It's a leftover from my notes